ideascube/serveradmin/tests/__init__.py

```diff
@@ -0,0 +1,59 @@
+try:
+    from NetworkManager import NM_DEVICE_TYPE_ETHERNET, NM_DEVICE_TYPE_WIFI
+
+except Exception:
+    # Use hard-coded values for the tests, these are constants anyway
+    NM_DEVICE_TYPE_ETHERNET = 1
+    NM_DEVICE_TYPE_WIFI = 2
+
+
+class NMAccessPoint(object):
+    def __init__(self, ssid, strength, wpa_flags):
+        self.Ssid = ssid
```

Comment:
Yeah, I'm mocking the whole NetworkManager dbus API, replacing some objects by these.

Therefore, I have to use the same API they export, and that means capitalized properties.